pkg/prowgen/prowgen_tests.go

```diff
@@ -47,7 +47,7 @@ func DiscoverTests(r Repository, openShiftVersion string, cronOverride *string)
 							LiteralTestStep: &cioperatorapi.LiteralTestStep{
 								As:       "test",
 								From:     "src",
-								Commands: fmt.Sprintf("make %s", test.Command),
+								Commands: fmt.Sprintf("${SKIP_MESH_AUTH_POLICY_GENERATION}=true make %s", test.Command),
```

Comment:
Shouldn't it be `SKIP_MESH_AUTH_POLICY_GENERATION=true make ...` ?

Reply:
Yeah my bad.